lib/src/components/NxFormSelect/types.ts

```diff
@@ -9,11 +9,11 @@ import { SelectHTMLAttributes } from 'react';
 
 export interface StateProps {
   value: string | number;
-  isPristine: boolean;
+  isPristine?: boolean | null;
```

Comment:
This can probably be removed entirely

Reply:
> This can probably be removed entirely

Removed it and also updated the propTypes.
I tried removing the propTypes entirely since that was the only attribute but it erroring out because of classNames.
Adding the value propType fixed it. Let me know what you think:
326a216

Reply:
> > > Removed it and also updated the propTypes.
> > > I tried removing the propTypes entirely since that was the only attribute but it erroring out because of classNames.
> > > Adding the value propType fixed it. Let me know what you think:
> > > [326a216](https://github.com/sonatype/sonatype-react-shared-components/commit/326a216be0343c14e875d4d94d34e4b89f057be9)
> > 
> > 
> > Seems to work like this: #532
> 
> Oh sorry, I mistook eslint error for type error: I'm getting `'className' is missing in props validation`
> 
> Adding `props: Props` to NxFormSelect fixed it. Thanks going to merge and update the PR.

Updated: 1501540e78d886d857f4b62a90ea8a73b332fb9c